backend/src/starknet.ts

```diff
@@ -19,7 +19,14 @@ export const getStarknetId = async (
     constants.StarknetChainId.SN_MAIN
   );
 
-  const domain = await starknetIdNavigator.getStarkName(address);
+  let domain;
+  try {
+    domain = await starknetIdNavigator.getStarkName(address);
+  } catch (error) {
+    console.error(error);
+    return null;
+  }
+
   const id = await starknetIdNavigator.getStarknetId(domain);
```

Comment:
It crashes at getStarknetId. Please properly reproduce the bug, fix it, **test it** and verify that it doesn't appear again. Now all the components are in place – contract, FE, BE.

Reply:
And, why is this treated as an error (sent to stderr)? It is not.

Reply:
@tensojka I've fixed the bug and added tests. Please review at your convenience